src/caret_analyze/plot/bokeh/callback_sched.py

```diff
@@ -132,10 +132,15 @@ def get_cbg_and_name(
     elif (isinstance(target, Path)):
         callback_groups = UniqueList()
         for comm in target.communications:
-            for cbg in comm.publish_node.callback_groups:
+            callback_groups = UniqueList(comm.publish_node.callback_groups or [])
+            for cbg in callback_groups:
                 callback_groups.append(cbg)
-        for cbg in target.communications[-1].subscribe_node.callback_groups:
+
+        callback_groups = UniqueList(
+                target.communications[-1].subscribe_node.callback_groups or [])
+        for cbg in callback_groups:
```

Comment:
callback_groups seems be unintendly overrided.
```suggestion
        for cbg in target.communications[-1].subscribe_node.callback_groups or []:
```